src/aat/resources/features/F-106/F-106.feature

```diff
@@ -37,3 +37,28 @@ Feature: F-106: Update Supplementary Data
     And it is submitted to call the [Update Supplementary Data] operation of [CCD Data Store api],
     Then a positive response is received,
     And the response has all the details as expected.
+
+  @S-608 @elasticsearch @Ignore
+  Scenario: Must return the updated supplementary data values from Data store and search through elastic search
+    Given a case [C1, which has just been] created as in [F106_Case_Data_Create_C1],
+    And a successful call [by Dil to update supplementary_data] as in [F-106_Set_Supplementary_Data_C1],
+    And a wait time of [5] seconds [to allow for Logstash to index the case just created],
+    And a user with [a valid profile],
+    And the request [is configured to search for supplementary data from previously updated case],
```

Comment:
Might suggest a small tweak to the bdd here `the request [is configured to search for the previously created case by the updated supplementary data value]`